elasticsearch/connection/base.py

```diff
@@ -200,5 +243,5 @@ def _get_api_key_header_val(self, api_key):
         """
         if isinstance(api_key, (tuple, list)):
             s = "{0}:{1}".format(api_key[0], api_key[1]).encode('utf-8')
-            return "ApiKey " + base64.b64encode(s).decode('utf-8')
+            return "ApiKey " + binascii.b2a_base64(s).rstrip(b"\r\n").decode('utf-8')
```

Comment:
It's a bummer that Python 2 doesn't have a `newline` kwarg for `b2a_base64()` so we need to call `rstrip()` on the result.